src/graphql/shopQuery.graphql

```diff
@@ -1,6 +1,15 @@
 query {
   shop {
     currencyCode
+    paymentSettings {
```

Comment:
The only payment setting I would include is `enabledPresentmentCurrencies`. The other ones are not critical for a flow that uses web checkout.